From aff7ac92685c166cd5b4e229eed0e4f02133f8f9 Mon Sep 17 00:00:00 2001
From: Felix Hoehle <fhoehle@users.noreply.github.com>
Date: Wed, 4 Dec 2024 09:14:32 +0100
Subject: [PATCH] update minimal go version

---
 go.mod | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/go.mod b/go.mod
index 4b5afa0..2139114 100644
--- a/go.mod
+++ b/go.mod
@@ -1,6 +1,7 @@
 module github.com/getsops/sops/v3
 
-go 1.22
+go 1.22.7
+
 toolchain go1.22.9
 
 require (
-- 
2.44.0

